source/reference/kubectl-minio-plugin/kubectl-minio-delete.rst

```diff
@@ -67,3 +68,15 @@ The command supports the following flags:
 
    Defaults to ``minio-operator``. 
 
+.. mc-cmd:: --dangerous
+   :optional:
+
+   Safety flag to confirm deletion of the MinIO Operator and all tenants with :mc-cmd:`~kubectl minio delete --force`.
```

Comment:
If I recall correctly, this may also delete the PVC and PV underlying if the PVC retention isn't set to `retain` - https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaim-policy

Maybe worth warning here and confirming with someone on the K8s team that deleting tenants could trigger PVC cleaning.

Reply:
🤔 That could be an unexpected surprise. I'll follow up with k8s folks about it.

And in the meantime add an example for the other update.

Reply:
Pedro confirmed the PVC and PV aren't deleted, because finalizers. But added a note about setting the storage class retain policy to `Retain` for good measure.